misk-slack/src/main/kotlin/slack-api/SlackJson.kt

```diff
@@ -0,0 +1,152 @@
+@file:Suppress("PropertyName") // We use snake_case to match Slack's JSON.
+
+package `slack-api`
+
+/**
+ * Message posted to /api/chat.postMessage
+ *
+ * https://api.slack.com/methods/chat.postMessage
+ */
+data class PostMessageJson(
+  val channel: String,
+  val response_type: String? = "in_channel",
+  val blocks: List<Any>,
+)
+
+/**
+ * An envelope that contains text or elements.
+ *
+ * https://api.slack.com/reference/block-kit/blocks
+ */
+data class BlockJson(
+  val type: String,
+  val replace_original: Boolean? = null,
+  val block_id: String? = null,
+  val text: TextJson? = null,
+  val accessory: ButtonLinkAndValueJson? = null,
+  val elements: List<ButtonLinkAndValueJson>? = null,
+)
+
+/** https://api.slack.com/reference/block-kit/composition-objects#text */
+data class TextJson(
```

Comment:
I wonder if all of these suffixed with Json are necessary? The scoping with the `misk.slack` or `misk.slack.api` package should be sufficient to imply that these are Slack specific API fixtures.

I'd consider moving the API definitions (this file, the retrofit file etc. into a `misk.slack.api` for improved organization)